docs/README.md

```diff
@@ -0,0 +1,118 @@
+# SHARK User Guide
```

Comment:
I'd personally put this at `/docs/user-guide.md`, but I could also see a case for `/docs/users/README.md` or where you have it now at `docs/README.md`.

Organizing into `docs/users/` and `docs/developers/` gives us a nice way to direct users away from guides that are less actively maintained.

Reply:
Let's do an incremental step. I'll go with user-guide.md since that accomplishes the goal and is to understand. We can further restructure later as we get more developer-only and user-only content.